src/SophiApp/Localizations/IT.xaml

```diff
@@ -3,26 +3,26 @@
     <system:String x:Key="Conditions.NewVersion.HasNew">é disponibile una nuova versione di SophiApp.</system:String>
-    <system:String x:Key="Conditions.NewVersion.OpenGitHub">Apri la repository su GitHub</system:String>
-    <system:String x:Key="Conditions.NoRebootRequired.RebootPending">Il Pc sta per essere riavviato.</system:String>
-    <system:String x:Key="Conditions.NoRebootRequired.PleaseReboot">Si prega di riavviare il PC prima di eseguire la SophiApp.</system:String>
-    <system:String x:Key="Conditions.OneInstanceOnly">Non eseguire più istanze di SophiApp contemporaneamente.</system:String>
-    <system:String x:Key="Conditions.OsBuildVersion.NotCompatible">SophiApp non è compatibile con la tua versione di Windows.</system:String>
+    <system:String x:Key="Conditions.NewVersion.OpenGitHub">Apri repository GitHub</system:String>
+    <system:String x:Key="Conditions.NoRebootRequired.RebootPending">Il computer sta per essere riavviato.</system:String>
+    <system:String x:Key="Conditions.NoRebootRequired.PleaseReboot">Prima di eseguire la SophiApp riavvia il computer.</system:String>
+    <system:String x:Key="Conditions.OneInstanceOnly">Non eseguire contemporaneamente più istanze di SophiApp.</system:String>
+    <system:String x:Key="Conditions.OsBuildVersion.NotCompatible">SophiApp non è compatibile con questa versione di Windows.</system:String>
     <system:String x:Key="Conditions.OsBuildVersion.Supported">SophiApp supporta Windows 10 2004/20H2/21H1/21H2,</system:String>
     <system:String x:Key="Conditions.OsBuildVersion.SupportedInsider">Windows 11 21H2, 22H2 &amp; Windows 11 Insider Preview.</system:String>
-    <system:String x:Key="Conditions.OsFilesNotCorrupted.IsCorrupted">Alcuni componenti di Windows potrebbero essere rotti.</system:String>
-    <system:String x:Key="Conditions.OsBuildVersion.RunUpdate">Esegui Windows Update per aggiornare il sistema operativo all'ultima versione.</system:String>
-    <system:String x:Key="Conditions.OsFilesNotCorrupted.ReinstallOs">Si prega di reinstallare il sistema operativo.</system:String>
-    <system:String x:Key="Conditions.OsIsInfected">Il tuo sistema potrebbe essere infettato per una backdoor in Win 10 Tweaker.</system:String>
-    <system:String x:Key="Conditions.OsIsInfected.More">Scopri di più:</system:String>
+    <system:String x:Key="Conditions.OsFilesNotCorrupted.IsCorrupted">Alcuni componenti di Windows potrebbero essere danneggiati.</system:String>
+    <system:String x:Key="Conditions.OsBuildVersion.RunUpdate">Esegui Windows Update per aggiornare il sistema operativo alla versione più recente.</system:String>
+    <system:String x:Key="Conditions.OsFilesNotCorrupted.ReinstallOs">Reinstalla il sistema operativo.</system:String>
+    <system:String x:Key="Conditions.OsIsInfected">Questo sistema potrebbe essere stato infettato tramite una backdoor in Win 10 Tweaker.</system:String>
```

Comment:
option 1 essere infetto da una backdor
opt 2 essere stato infettato da una backdor

Reply:
opt1 for me is not italian for the verb.
opt2 is the right form for verb. opt2 is already set.

Reply:
Is it ok?

Reply:
ok